### ISSUE TITLE
[flutter_tools] RPCError: streamListen: (-32601) The stream `ToolEvent` is not supported on web devices

### ISSUE THREAD

**Issue by christopherfujino** [flutter/flutter#124543]
On `beta/3.9.0-0.2.pre`
Command `flutter debug-adapter`

`package:dds` at `2.7.6` https://github.com/flutter/flutter/blob/0d9a0207adb0b9991f21d03e574a39b3a23f9842/packages/flutter_tools/pubspec.yaml#L13
`package:vm-service` at `11.2.0` https://github.com/flutter/flutter/blob/0d9a0207adb0b9991f21d03e574a39b3a23f9842/packages/flutter_tools/pubspec.yaml#L58

```
RPCError: streamListen: (-32601) The stream `ToolEvent` is not supported on web devices
at new _OutstandingRequest(vm_service.dart:1816)
at VmService._call(vm_service.dart:2360)
at VmService.streamListen(vm_service.dart:2307)
at DartDebugAdapter.connectDebugger(dart.dart:676)
at <asynchronous gap>(async)
at FlutterDebugAdapter._connectDebugger(flutter_adapter.dart:378)
at <asynchronous gap>(async)
at FlutterDebugAdapter._handleDebugPort(flutter_adapter.dart:443)
at <asynchronous gap>(async)
```

**Comment by christopherfujino** [comment]
cc @DanTup

**Comment by DanTup** [comment]
This issue was fixed by https://github.com/dart-lang/webdev/pull/2011. Based on the dates of dwds releases, I think the fix only made it into dwds 18.0.1 but the version noted above seems to be using 18.0.0.

I only [re-enabled the SDK DAP](https://github.com/Dart-Code/Dart-Code/commit/fef48bc80a13a8637760ea9f99c458bcec5de0cc) for >= 3.9.0-14 so this might be someone that's explicitly opted-in (or on an outdated VS Code extension or something)?

In any case, I expect the issue will go away with the next beta and I don't think there's anything to do here.

**Comment by christopherfujino** [comment]
@DanTup Thanks, and yeah, the next release will be with 19.0.0, so I think we're good to go: https://github.com/flutter/flutter/blob/flutter-3.10-candidate.1/packages/flutter_tools/pubspec.yaml
